### ISSUE TITLE
.near archer int list || .near list archer int ... List issue

### ISSUE THREAD

**Issue by hamilton1977** [STFCcommunity/botfeedback#17]
Mode:
list            - list of systems within the criteria

----------

List doesn't change the output.

**Comment by hamilton1977** [comment]
okay.  confusing at 1st glance.

**Comment by Iliasgx** [comment]
Yes, the list mode was copied from the system command, which used the same code to find a result. But since the nearest command will only output a list, we may remove that mode from the nearest help command. 👍

**Comment by hamilton1977** [comment]
i agree.  if there isn't a reason for a toggle then its just as leads to saying "this is a bug." Its like that light switch that doesn't do anything in the room because know one knows which plug its wired to. So,  it's just a busted light switch.

**Comment by WebSpider** [comment]
This will be removed from help text in the next update
